frontend/src/components/SearchBar.js

```diff
@@ -0,0 +1,26 @@
+import { TextField, styled } from '@mui/material';
+
+const SearchBar = () => {
+  const MyTextField = styled(TextField)({
+    '& .MuiOutlinedInput-root': {
+      backgroundColor: 'var(--main-light-secondary-2)',
+      '& fieldset': {
+        borderColor: 'var(--main-light-secondary)',
+      },
+      '&:hover fieldset': {
+        borderColor: 'var(--main-light-secondary)',
+        border: '1px solid'
+      },
+      '&.Mui-focused fieldset': {
+        borderColor: 'var(--main-light-secondary)',
```

Comment:
- Reformat `color` property by taking reference from my branch: `feature/left-drawer`
- For example: 
- `backgroundColor: 'var(--main-light-secondary-2)'` will be replaced by `backgroundColor: COLORS.<vaiable_name>`